.env.local

```
@@ -0,0 +1,2 @@
+API_BASE_URL=https://chinguchingu.kro.kr
+NEXT_PUBLIC_API_URL=https://chinguchingu.kro.kr
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)